types/sinon/sinon-tests.ts

```diff
@@ -215,7 +215,6 @@ function testClock() {
     clock.reset();
     clock.runMicrotasks();
     clock.runToFrame();
-    clock.uninstall();
```

Comment:
clocks don't have an `uninstall` method anymore?

Reply:
Apparently, yes. Tests were failing without removing this line.

Reply:
their docs and the types still have it, it seems. what failure do you get when its there? we should try fix it